extensions/integration-tests/src/notebook.test.ts

```diff
@@ -31,75 +48,46 @@ if (context.RunTest) {
-		// 	console.log('Start Python3 NB test');
-		// 	let notebook = await openNotebook(pySparkNotebookContent, pythonKernelMetadata);
-		// 	let cellOutputs = notebook.document.cells[0].contents.outputs;
-		// 	console.log('Got cell outputs');
-		// 	let result = (<azdata.nb.IExecuteResult>cellOutputs[0]).data['text/plain'];
-		// 	assert(result === '2', `Expected: 2, Actual: ${result}`);
-		// 	console.log('Python3 NB done');
-		// });
-
-		// test('Clear all outputs - Python3 notebook ', async function () {
-		// 	let notebook = await openNotebook(pySparkNotebookContent, pythonKernelMetadata);
-		// 	//Check if at least one cell with output
-		// 	let cellWithOutputs = notebook.document.cells.find(cell => cell.contents && cell.contents.outputs && cell.contents.outputs.length > 0);
-		// 	console.log("Before clearing cell outputs");
-		// 	if (cellWithOutputs) {
-		// 		let clearedOutputs = await notebook.clearAllOutputs();
-		// 		let cells = notebook.document.cells;
-		// 		cells.forEach(cell => {
-		// 			assert(cell.contents && cell.contents.outputs && cell.contents.outputs.length === 0, `Expected Output: 0, Actual: '${cell.contents.outputs.length}'`);
-		// 		});
-		// 		assert(clearedOutputs, 'Outputs of all the code cells from Python notebook should be cleared');
-		// 		console.log("After clearing cell outputs");
-		// 	}
-		// 	assert(cellWithOutputs === undefined, 'Could not find notebook cells with outputs');
-		// });
-
 		test('Clear all outputs - SQL notebook ', async function () {
-			let notebook = await openNotebook(sqlNotebookContent, sqlKernelMetadata);
-			let cellWithOutputs = notebook.document.cells.find(cell => cell.contents && cell.contents.outputs && cell.contents.outputs.length > 0);
-			console.log('Before clearing cell outputs');
-			if (cellWithOutputs) {
-				let clearedOutputs = await notebook.clearAllOutputs();
-				let cells = notebook.document.cells;
-				cells.forEach(cell => {
-					assert(cell.contents && cell.contents.outputs && cell.contents.outputs.length === 0, `Expected cell outputs to be empty. Actual: '${cell.contents.outputs}'`);
-				});
-				assert(clearedOutputs, 'Outputs of all the code cells from SQL notebook should be cleared');
-				console.log('After clearing cell outputs');
-			}
-			else {
-				throw new Error('Could not find notebook cells with outputs');
-			}
+			let notebook = await openNotebook(sqlNotebookContent, sqlKernelMetadata, this.test.title);
+			await verifyClearAllOutputs(notebook);
 		});
 
+		if (process.env.RUN_PYTHON3_TEST === '1') {
+			test('Python3 notebook test', async function () {
+				let notebook = await openNotebook(pySparkNotebookContent, pythonKernelMetadata, this.test.title);
+				let cellOutputs = notebook.document.cells[0].contents.outputs;
+				console.log('Got cell outputs');
+				let result = (<azdata.nb.IExecuteResult>cellOutputs[0]).data['text/plain'];
+				assert(result === '2', `Expected: 2, Actual: ${result}`);
```

Comment:
Expected 2 what? Try to be a little more descriptive for failure messages like this - it helps debug test failures a lot easier